### DISCUSSION TITLE
How does `yarn link` work?

### DISCUSSION THREAD

**Discussion by skoenfaelt** [yarnpkg/berry#6571] [Q&A] [answered]
What I expect to work

```sh
cd /path/to/my-local-package
yarn link
```

```sh
cd /path/to/my-project
yarn link "my-local-package"
```

Now "my-local-package" should be a dependency inside "my-project" but instead i got this error "Usage Error: Invalid destination 'my-local-package'; Can't link the project to itself"

Is it because my local package has a scope prefix? `yarn link @my-scope/shared-components` is not working.

-- I do not work with workspaces --

For example with Bun this works just as expected.

**Comment by merceyz** [comment, marked as answer]
I suggest taking a look at the documentation for the command at https://yarnpkg.com/cli/link or by running
```sh
yarn link --help
```

In your case you would run 
```sh
yarn link /path/to/my-local-package
```
from `/path/to/my-project`.

**Reply by skoenfaelt** [reply]
Okay check.
I was worried i have to realign my codebase to use workspaces.
